react-native-gutenberg-bridge/index.js

```diff
@@ -10,6 +10,12 @@ const isIOS = Platform.OS === 'ios';
 
 const gutenbergBridgeEvents = new NativeEventEmitter( RNReactNativeGutenbergBridge );
 
+export const mediaSources = {
+	deviceLibrary: 'DEVICE_MEDIA_LIBRARY',
+	deviceCamera: 'DEVICE_CAMERA',
+	siteMediaLibrary: 'SITE_MEDIA_LIBRARY',
+};
```

Comment:
@SergioEstevao - I decided to set the media source constants bundled in an object.
Anyway, at some point we will probably move all this to the parent apps.